command/cp.go

```diff
@@ -280,7 +288,8 @@ func (c Copy) Run(ctx context.Context) error {
 		}
 	}()
 
-	isBatch := srcurl.HasGlob()
+	// use raw flag to handle glob operations.
```

Comment:
```suggestion
	// use --raw flag to prevent glob operations.
```
or you can remove the comment entirely.

Reply:
changed the comment as you suggested.